src/less/core.less

```diff
@@ -74,6 +74,11 @@
     width: 100%;
   }
 }
+
+.rw-now {
+  border: 1px solid #adadad;
```

Comment:
Yep - very good suggestion, I felt the same way (but, lazy). Give me a tick to throw it in!